layers/meta-balena-beaglebone/recipes-core/images/balena-image.bbappend

```diff
@@ -6,3 +6,14 @@ IMAGE_ROOTFS_SIZE:beaglebone-ai64 = "733184"
 IMAGE_INSTALL:append:beaglebone = " bb-org-overlays fix-mmc-bbb bb-wl18xx-bluetooth bb-wl18xx-wlan0"
 IMAGE_INSTALL:append:beaglebone-ai64 = " mmc-utils vxd-dec-fw"
 IMAGE_INSTALL:append:beaglebone-pocket = " boot-scripts"
+IMAGE_INSTALL:append:beagleplay = " mmc-utils"
+IMAGE_INSTALL:append:beagleplay-k3r5 = " mmc-utils"
+
+IMAGE_INSTALL:remove:beagleplay = " bb-wl18xx-btwl"
+IMAGE_INSTALL:remove:beagleplay-k3-r5 = " bb-wl18xx-btwl"
+
+IMAGE_ROOTFS_MAXSIZE:beagleplay = "1966080"
+IMAGE_ROOTFS_MAXSIZE:beagleplay-k3r5 = "1966080"
+
+IMAGE_ROOTFS_SIZE:beagleplay = "1966080"
```

Comment:
Why use so much space? What is the actual size of the generated rootfs?

Reply:
I recall that I needed to increase the rootfs_size to get the yocto build to run to completion. I recall that I increased the rootfs in two or three steps (doubling and then tripling it) before things worked.

Reply:
Like for above, to check how big the rootfs is you can look at `tmp/work/<board>-poky-linux/balena-image/1.0-r0/rootfs` in your build directory.

Reply:
@floion - here's the output from the `du -h` command:

```
ubuntu@ip-172-31-16-228:~/balena-beaglebone/build/tmp/work/beagleplay-poky-linux/balena-image/1.0-r0/rootfs$ du -h --max-depth=1
4.0K    ./proc
72M     ./boot
4.0K    ./sys
62M     ./resin-boot
16K     ./home
2.8M    ./sbin
4.0K    ./media
1.2G    ./lib
4.0K    ./tmp
149M    ./usr
4.0K    ./srv
4.0K    ./run
4.0K    ./dev
12K     ./quirks
4.0K    ./resin-data
72K     ./var
2.6M    ./etc
28K     ./mnt
6.0M    ./bin
1.4G    .
```

Looks like the bulk of the usage is in the `lib` subfolder.

Reply:
Which in turn has a (very) large `firmware` folder:

```
ubuntu@ip-172-31-16-228:~/balena-beaglebone/build/tmp/work/beagleplay-poky-linux/balena-image/1.0-r0/rootfs/lib$ du -h --max-depth=1
7.3M    ./systemd
1.1G    ./firmware
12K     ./resolvconf
38M     ./modules
12K     ./modprobe.d
12K     ./depmod.d
576K    ./udev
1.2G    .
```